go.mod

```diff
@@ -74,3 +74,5 @@ require (
 )
 
 go 1.13
+
+replace github.com/pingcap/parser => /Users/zhangyuanjia/Workspace/go/src/github.com/pingcap/parser
```

Comment:
Remove this line.

Use command:

```
go mod edit -replace github.com/pingcap/tidb/parser=github.com/qw4990/parser@config-priv
```

And try again.

Reply:
Thx, it is fixed.